### PR TITLE
feat:DX-1085; set default search query for ext search

### PR DESCRIPTION
**Pull request by okorie2**

### CODE REVIEW

File: services/external_sources/who.py [hunk 1]
```diff
@@ -27,6 +28,7 @@ def who_search(query, owner, limit=5, prev=0):
     :param prev: The number of results to skip
     :return: A list of dicts in the form of EXTERNAL_DATASET_FORMAT
     """
+
```
Comment (sylvanr) [inline services/external_sources/who.py:31]: Do remove the additional empty line!

File: services/external_sources/who.py [hunk 2]
```diff
@@ -53,8 +55,6 @@ def _who_search_indicators(query, skip, top):
     :skip: The number of results to skip
     :param top: The number of results to return
```
Comment (sylvanr) [inline services/external_sources/who.py:57]: remove the empty line at the end of the comment

File: services/external_sources/external_sources.py [hunk 3]
```diff
@@ -11,6 +11,7 @@
 logger = logging.getLogger(__name__)
 load_dotenv()
 
+DEFAULT_SEARCH_TERM = "world population"
```
Comment (sylvanr) [inline services/external_sources/external_sources.py:15]: two newlines between function definitions and other content such as a constant. you can check with `flake8 services` to see any of those issues

File: services/external_sources/kaggle.py [hunk 4]
```diff
@@ -48,10 +48,8 @@ def kaggle_search(query, owner, limit=5, prev=0):
     logger.debug(f"Searching kaggle for query: {query}")
     res = []
     try:
-        if query == "":
-            command = "kaggle datasets list --file-type csv --csv --max-size 5000000"
-        else:
-            command = f"kaggle datasets list --file-type csv -s {query} --csv --max-size 5000000"
+
```
Comment (sylvanr) [inline services/external_sources/kaggle.py:51]: remove the empty line

File: services/external_sources/kaggle.py [hunk 5]
```diff
@@ -48,10 +48,8 @@ def kaggle_search(query, owner, limit=5, prev=0):
...
+        command = f'''kaggle datasets list --file-type csv -s "{query}" --csv --max-size 5000000'''
```
Comment (sylvanr) [inline services/external_sources/kaggle.py:52]: why tripple quotes here?

**Review by sylvanr — CHANGES_REQUESTED** [review]
just some quality control stuff, do fix and will merge after

File: services/external_sources/kaggle.py (hunk 5 above)
Comment (okorie2) [reply services/external_sources/kaggle.py:52]: to allow me wrap query in quote

**Review by sylvanr — APPROVED** [review]

**Comment by sylvanr** [comment]
Rebased on latest main

**Pull request merged**